test/uk/gov/hmrc/timetopayproxy/models/CreatePlanRequestSpec.scala

```diff
@@ -291,6 +291,20 @@ class CreatePlanRequestSpec extends AnyWordSpec with Matchers with CreatePlanReq
       }
     }
 
+    "fail decoding if addressPostcode is empty" in {
+      import play.api.libs.json._
```

Comment:
Why is this test added? And why are we importing play json libraries here?

Reply:
Ah, coverage!

Reply:
Coverage! But you're right about the redundant import.